woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1340,7 +1342,7 @@ def test_dataframe_methods_on_accessor_inplace(sample_df):
 
 
 def test_dataframe_methods_on_accessor_returning_series(sample_df):
-    schema_df = sample_df.copy()
+    schema_df = sample_df[['id', 'age', 'is_registered']]
```

Comment:
Is there a reason we had to make this change for this test?

Reply:
yes, this is related to the `FutureWarning: Dropping of nuisance columns in DataFrame reductions (with 'bool_only=None') is deprecated; in a future version this will raise TypeError. Select only valid columns before calling the reduction.` warning. Seems like reductions like `min`, `max,` and `all` that reduce a DataFrame to a Series now should only be called on valid columns.

Reply:
Got it. I think we could include the `signup_date` column as well, but not a big deal either way.